app/models/form.rb

```diff
@@ -8,10 +9,14 @@ def start_page
 
   def make_live!
     update!(live_at: Time.zone.now)
+
+    published_forms.create!(version: to_json(include: [:pages])) if FeatureService.enabled?(:draft_live_versioning)
```

Comment:
@danworth no problem, everything is still behind feature flag because there are still some ui work in forms-admin that needs to be completed before we should be using this.